doc/man3/OSSL_PROVIDER.pod

```diff
@@ -14,6 +14,7 @@ OSSL_PROVIDER_add_builtin - provider routines
 
  OSSL_PROVIDER *OSSL_PROVIDER_load(OPENSSL_CTX *, const char *name);
  int OSSL_PROVIDER_unload(OSSL_PROVIDER *prov);
+ int OSSL_PROVIDER_available(OPENSSL_CTX *, const char *name);
```

Comment:
This is missing from the NAME section (hence the travis failure)

Reply:
Fixed